### ISSUE TITLE
mbsync will be getting XDG support in 1.5

### ISSUE THREAD

**Issue by peterhoeg** [nix-community/home-manager#3317]
### Description

hm currently drops `.mbsyncrc` into `$HOME` but with the upcoming 1.5, it will gain proper XDG support:

https://sourceforge.net/p/isync/feature-requests/14/

**Comment by rycee** [comment]
Cool 🙂 PR very welcome after the new version is available in Nixpkgs unstable.

**Comment by peterhoeg** [comment]
1.5 hasn't hit nixpkgs yet.

**Comment by sumnerevans** [comment]
Is there a Nixpkgs PR to track?

**Comment by peterhoeg** [comment]
No, because the reason it isn’t in nixpkgs is because upstream hasn’t released it yet… ;-)

**Comment by peterhoeg** [comment]
1.5 has not been released yet.

**Comment by nekowinston** [comment]
No new release yet.

**Comment by felipeqq2** [comment]
1.5 was [released](https://sourceforge.net/projects/isync/files/isync/1.5.0/)! :tada: 

Related: https://github.com/NixOS/nixpkgs/pull/331813

**Comment by teto** [comment]
done
